addons/sale_project/data/project_demo.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <function model="project.project" name="write">
+            <value model="project.project" search="[('id', '=', ref('project.project_home_construction'))]"/>
+            <value eval="{
+                'allow_billable': 'True',
+            }"/>
+        </function>
+
+        <!-- Task in project product -->
+        <record id="product_product_painting" model="product.product">
+            <field name="name">Painting</field>
+            <field name="categ_id" ref="product.product_category_construction"/>
+            <field name="list_price">1000</field>
```

Comment:
All prices should be in double (1000.00), for consistency :see_no_evil:

Reply:
watch for all occurrences in this pr, please

Reply:
1- https://github.com/odoo/odoo/blob/master/addons/product/models/product_template.py#L50 its 'consu' only. :see_no_evil: 
2 - got it.
Thanks.

Reply:
Whoops you're right :ok_hand: